pkg/apiserver/database/gorm/odata.go

```diff
@@ -1106,6 +1106,24 @@ var schemaMetas = map[string]odatasql.SchemaMeta{
 			"state":              odatasql.FieldMeta{FieldType: odatasql.StringFieldType},
 		},
 	},
+	providerSchemaName: {
+		Table: "assets",
```

Comment:
This should be providers, and there will need to be changes in database.go to add the new table as well as add indexes for the ID field, and any other commonly queried fields.